From: Helmut Grohne <helmut@subdivi.de>
Date: Mon, 6 Mar 2017 22:29:23 -0500
Subject: use PKG_PROG_PKG_CONFIG correctly

Using $PKG_CONFIG allows PKG_PROG_PKG_CONFIG to add $ac_tool_prefix which is
crucial for cross compilation.
---
 configure.ac | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index b08bfe60..559a6f67 100644
--- a/configure.ac
+++ b/configure.ac
@@ -55,8 +55,8 @@ dnl
 dnl Calls pkg-config --static
 dnl
 AC_DEFUN([PKG_STATIC], [
-  if AC_RUN_LOG([pkg-config --exists --print-errors "$2"]); then
-    $1=`pkg-config --libs --static "$2"`
+  if AC_RUN_LOG([$PKG_CONFIG --exists --print-errors "$2"]); then
+    $1=`$PKG_CONFIG --libs --static "$2"`
     AC_SUBST([$1])
   else
     AC_MSG_ERROR([pkg-config description of $2, needed for static build, is not available])
@@ -173,8 +173,8 @@ PKG_STATIC(ZLIB_LIBS_STATIC, [zlib])
 # Our udev rule gives us the friendly dm names but isn't required (or valid)
 # on earlier releases.
 UDEVDIR=
-if pkg-config udev --atleast-version 190; then
-	UDEVDIR="$(pkg-config udev --variable=udevdir)"
+if $PKG_CONFIG udev --atleast-version 190; then
+	UDEVDIR="$($PKG_CONFIG udev --variable=udevdir)"
 fi
 AC_SUBST(UDEVDIR)
 
